mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule Nerves.MixProject do
     [
       {:castore, "~> 0.1"},
       {:elixir_make, "~> 0.6", runtime: false},
-      {:jason, "~> 1.2", optional: true},
+      # {:jason, "~> 1.2", optional: true},
```

Comment:
Why is the `:jason` dependency commented out?

Reply:
O derp 🤦 because i commented out in order to test the error happens